addons/bus/static/src/js/services/bus_service.js

```diff
@@ -58,29 +58,33 @@ var BusService =  CrossTab.extend(ServicesMixin, {
     /**
      * Send a notification, and notify once per browser's tab
      *
-     * @param {string} title
-     * @param {string} content
+     * @param {object} options
+     * @param {string} options.title
+     * @param {string} options.message
+     * @param {string} [options.type='warning'] 'info', 'success', 'warning', 'danger' or ''
+     * @param {boolean} [options.hasHeaderIcon]
+     * @param {string} [options.titleColor]
      * @param {function} [callback] if given callback will be called when user clicks on notification
      */
-    sendNotification: function (title, content, callback) {
+    sendNotification: function (options, callback) {
```

Comment:
since you have diff on the line already, let's shorten it
```suggestion
    sendNotification(options, callback) {
```